plugins/inputs/gnmi/gnmi.go

```diff
@@ -72,6 +74,9 @@ type Subscription struct {
 	// Duplicate suppression
 	SuppressRedundant bool              `toml:"suppress_redundant"`
 	HeartbeatInterval internal.Duration `toml:"heartbeat_interval"`
+
+	// Tag-only identification
+	TagOnly bool `toml:"tag_only"`
```

Comment:
not sure I understand the utility of this. It's checked only in one place.

Reply:
Serves the same function as [isTag](https://github.com/influxdata/telegraf/blob/d369003912e6568aeb43c8d56207e407550afa01/plugins/inputs/snmp/snmp.go#L231-L232) in SNMP plugin -- to identify that this specific reading shouldn't be emitted to outputs, but stored to apply to other telemetry flows